internal/errors/code.go

```diff
@@ -27,6 +27,21 @@ const (
 	InvalidFieldMask Code = 103 // InvalidFieldMask represents an invalid field mast for an operation
 	EmptyFieldMask   Code = 104 // EmptyFieldMask represents an empty field mask for an operation
 
+	// PasswordTooShort results from attempting to set a password which is to short.
+	PasswordTooShort Code = 200
+
+	// PasswordUnsupportedConfiguration results from attempting to perform an
```

Comment:
Now might be a good time to expand on what it means for a password config to be unsupported vs. invalid.

Unsupported means boundary is unable to support the password config type (right now any that is not argon2)
Invalid means any supported configuration that does not have parameters which boundary is able to use.